src/btree/row_modify.c

```diff
@@ -252,8 +252,9 @@ __wt_row_modify(WT_CURSOR_BTREE *cbt, const WT_ITEM *key, const WT_ITEM *value,
     inserted_to_update_chain = true;
 
     /* If the update was successful, add it to the in-memory log. */
-    if (logged && modify_type != WT_UPDATE_RESERVE) {
-        WT_ERR(__wt_txn_log_op(session, cbt));
+    if (logged) {
```

Comment:
Why is this split now? Is there work we have to do for reserve types? If so can we update the comment above?